daemon/manager/manager.go

```diff
@@ -76,24 +75,17 @@ func Execute() {
 	log.Info("controller Start successful")
 	defer controller.Stop()
 
-	if bpf.GetConfig().EnableKmeshWorkload {
-		if err = controller_workload.Start(); err != nil {
+	if bpf.GetConfig().EnableKmesh {
+		if err = dump.StartServer(); err != nil {
```

Comment:
We need to start this server no matter kmesh or workload mode

Reply:
Currently, the workload mode does not support dumping data.